src/App.js

```diff
@@ -3,20 +3,21 @@ import title from './craftybeer-title.png'
 import './App.css';
 import Search from './components/Search'
 import { Routes, Route, Link } from 'react-router-dom'
+import BrewDetails from './BrewDetails';
+import Header from './components/Header'
 
 
 const App =() => {
   
   return (
     <>
-      <header className="header">
-        <Link to="/">
-          <img className ="title" src={title} 
-            alt="crafty beer title"/>
-        </Link>
-      </header> 
+      <Header />
       <main className='main-container'>
-        <Search />      
+        {/* <Search />       */}
+      <Routes>
+        <Route path='/' element={<Search />} />
```

Comment:
Are you planning on adding an error page for when the route is incorrect?? If so, Nate posted something in the private channel that has good insight on doing this with dynamic routes!